src/components/shared/SelectPeople.tsx

```diff
@@ -131,38 +161,39 @@ export function SelectPeopleUI(props: SelectPeopleUI) {
                     disabled={disabled}
                 />
             </Box>
-            {disabled ? (
-                undefined
-            ) : (
-                <>
-                    <Box display="flex">
-                        {listAfterSearch.length > 0 && SelectAllButton}
-                        {selected.length > 0 && SelectNoneButton}
-                    </Box>
-                    <Box flex={1}>
-                        <List dense>
-                            {listBeforeSearch.length > 0 && listBeforeSearch.length === 0 && (
-                                <ListItem>
-                                    <ListItemText primary={geti18nString('not_found')} />
-                                </ListItem>
-                            )}
-                            {listAfterSearch.map(PeopleListItem)}
-                        </List>
-                    </Box>
-                </>
-            )}
-        </>
+            <Box display="flex">
+                {showSelectAll && SelectAllButton}
+                {showSelectNone && SelectNoneButton}
+            </Box>
+            <Box flex={1}>
+                <List dense>
+                    {listBeforeSearch.length > 0 && listBeforeSearch.length === 0 && (
+                        <ListItem>
+                            <ListItemText primary={geti18nString('not_found')} />
+                        </ListItem>
+                    )}
+                    {listAfterSearch.map(PeopleListItem)}
```

Comment:
Why can't I select person with `nickname`? Cause we display person with `nickname` but can't search by that.